pkg/util/etcd/etcd.go

```diff
@@ -0,0 +1,15 @@
+package etcd
+
+import (
+	"fmt"
+
+	"github.com/sapcc/kubernikus/pkg/apis/kubernikus/v1"
+)
+
+const (
+	EtcdBackupStorageContainer = "etcd-backup-%s-%s"
+)
+
+func DefaultStorageContainer(kluster *v1.Kluster) string {
+	return fmt.Sprintf(EtcdBackupStorageContainer, kluster.GetName(), kluster.GetUID())
```

Comment:
`GetName()` returns the full name of the kluster CRD ($NAME-$PROJECTID). As the storage container is created inside the project I think we can safely remove the project id from the container name and just go with $NAME-$UID
```suggestion
	return fmt.Sprintf(EtcdBackupStorageContainer, kluster.Spec.Name, kluster.GetUID())
```

Reply:
@databus23 I changed this.